### PR TITLE
Change scaling factor to 10^-5 if we run on skimmed data

### PR DESCRIPTION
**Pull request by chiarazampolli**
@matthias-kleiner , @catalinristea , let me know if this is fine. 
For now we can use a JDL variable, if we don't change tag for pp 2022 reco on skimmed data. 

Note that I don't know Marian's github account, I try with @mivanov.

### CODE REVIEW

**Comment by matthias-kleiner** [comment]
Hello @chiarazampolli 
didnt @miranov25 wanted to have 0.1? Maybe he can comment

**Comment by miranov25** [comment]
Yes. For the skimmed reconstruction production  we need factor 0.1

File: DATA/production/configurations/asyncReco/setenv_extra.sh [hunk 1]
```diff
@@ -561,7 +566,13 @@ if [[ $ALIEN_JDL_EXTRACTTIMESERIES == 1 ]]; then
   if [[ ! -z "$ALIEN_JDL_ENABLEUNBINNEDTIMESERIES" ]]; then
     export ARGS_EXTRA_PROCESS_o2_tpc_time_series_workflow="$ARGS_EXTRA_PROCESS_o2_tpc_time_series_workflow --enable-unbinned-root-output --sample-unbinned-tsallis --threads 1"
   fi
-  if [[ ! -z "$ALIEN_JDL_SAMPLINGFACTORTIMESERIES" ]]; then
+  if [[ $ON_SKIMMED_DATA == 1]] || [[ ! -z "$ALIEN_JDL_SAMPLINGFACTORTIMESERIES" ]] ; then
+    if [[ $ON_SKIMMED_DATA == 1]] ; then
+      SAMPLINGFACTORTIMESERIES=0.00001f
```
Comment (miranov25) [inline DATA/production/configurations/asyncReco/setenv_extra.sh:571]: For skimmed production shoul be 0.1

Comment (chiarazampolli) [reply DATA/production/configurations/asyncReco/setenv_extra.sh:571]: Sorry, you are definitely right, of course. I will fix it.

Comment (chiarazampolli) [reply DATA/production/configurations/asyncReco/setenv_extra.sh]: Done, can you please recheck that everything is in order? @matthias-kleiner , @miranov25

**Comment by miranov25** [comment]
I checked latest code. Looks fine.

**Comment by chiarazampolli** [comment]
Thanks @miranov25 , merged!

**Pull request merged**